packages/picasso-lab/src/Section/Section.tsx

```diff
@@ -13,6 +13,8 @@ import { Rotate180 } from '@toptal/picasso/utils/Transitions'
 
 import styles from './styles'
 
+type VariantType = 'block'
```

Comment:
We should have a fallback variant.

```suggestion
type VariantType = 'block' | 'default'
```

Reply:
To me, `block` has an implicit association with the block model in CSS. When I first looked at this PR, it was hard to fight this bias and learn it is not about this but about borders.

How about `type VariantType = 'bordered' | 'default'`? 
* it is also about some padding but the padding is needed due to border, so I believe `bordered` better describes this variant
* no confusion with block model
* in Picasso there are already components which have `bordered` variants, hence this adds some consistency